.github/workflows/config/.markdown-link-check/config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "ignorePatterns": [
+        {
+            "pattern": "https://docs.github.com"
+        },
+        {
+            "pattern": "https://guides.github.com"
+        },
+        {
+            "pattern": "https://help.github.com"
+        },
+        {
+            "pattern": "https://www.bottlecaps.de/rex/"
+        }
+  ],
+  "replacementPatterns": [
+        {
+            "pattern": "^../../issues",
+            "replacement": "https://github.com/usnistgov/metaschema/issues"
+        },
+        {
+            "pattern": "^../../projects",
+            "replacement": "https://github.com/usnistgov/metaschema/projects"
+        },
```

Comment:
Non-blocking: is Metaschema not using a [projects board](https://github.com/orgs/usnistgov/projects/44) that not a repo-based [projects (classic} board](https://github.com/usnistgov/metaschema/projects/1)? Is that mapping still applicable or appropriate?

Reply:
I'll update this.